python/ql/test/library-tests/frameworks/data/test.py

```diff
@@ -54,7 +54,7 @@ class Sub2Class (CommonTokens.Class):
 
 from testlib import ArgPos
 
-val = ArgPos.selfThing(arg, named=2)
+arg_pos = ArgPos(); val = arg_pos.self_thing(arg, named=2);
```

Comment:
This code looks a bit strange since it's not split into multiple lines... but it's also such a small nitpick.

Reply:
It was just to avoid updating the line numbers of all the lines below.  
That caused the many changes in the `.expected` file.